source/js/components/join/join.jsx

```diff
@@ -536,8 +557,7 @@ export default class JoinUs extends React.Component {
 
 JoinUs.defaultProps = {
   ctaHeader: `Protect the internet as a global public resource`,
-  ctaDescription: `Join our email list to take action and stay updated!`,
-  thankYouMessage: `If you haven’t previously confirmed a subscription to a Mozilla-related newsletter you may have to do so. <strong>Please check your inbox or your spam filter for an email from us.</strong>`,
```

Comment:
Removed as none of the signup form has its own customized thank you message (e.g., none of the `.join-us` has `data-thank-...` attribute)